mitxpro/settings.py

```diff
@@ -723,6 +741,13 @@ def get_all_config_keys():
     },
 }
 if FEATURES.get("COUPON_SHEETS"):
+    CELERY_BEAT_SCHEDULE["renew_file_watches"] = {
+        "task": "sheets.tasks.renew_file_watches",
+        "schedule": (
+            DRIVE_WEBHOOK_EXPIRATION_MINUTES - DRIVE_WEBHOOK_RENEWAL_PERIOD_MINUTES
+        )
+        * 60,
```

Comment:
`((60*24) - 30) * 60 == 84600` - that doesn't seem right?  Should be less than `DRIVE_WEBHOOK_EXPIRATION_MINUTES` (60*24)?

Reply:
These settings values are expressed in minutes (for convenience), but Celery task schedules are expected to be in seconds. The `60*24` is the number of minutes in a day and `30` is the number of minutes before that period that we want to try to refresh.

Reply:
👍